secretmanager/snippets/regional_samples/regional_quickstart.py

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env python
+
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+"""
+command line application and sample code for creating an accessing a secret.
+"""
+
+
+def regional_quickstart(_project_id: str, _location_id: str, _secret_id: str) -> None:
```

Comment:
Having two variables seems odd, consider following format:

```
def regional_quickstart(project_id: str, ...):

    # TODO (developer): uncomment project_id and assign a value.
    # project_id = "YOUR_PROJECT_ID"
```

Reply:
This was done to make it consistent like here : https://cloud.google.com/secret-manager/docs/create-secret-quickstart#secretmanager-quickstart-python
(The above is the documentation of global SM)

Reply:
Node example does a similar approach to ask the developer to uncomment. The current setup is an unfortunate tech debt - the samples team has no preference between the two styles, so I'd favor solutions that don't require more tech debt.